app/Dockerfile

```diff
@@ -28,6 +28,8 @@ COPY .coveragerc /src
 
 WORKDIR /
 
+RUN apt -y install curl
```

Comment:
@tristanhcole should this be here since it was added higher in 260?

Reply:
Yeah that’s an accident since was a fork of the other branch, will fix

Reply:
gone and done it from github :)